### PR TITLE
Added fill timetable buttons on Course List page

### PR DESCRIPTION
**Pull request by ctm-fl**

### CODE REVIEW

File: uniapp/src/page/UniTimetableMgt.Codeunit.al [hunk 1]
```diff
@@ -0,0 +1,71 @@
+codeunit 50000 "Uni Timetable Mgt."
+{
+    TableNo = "Uni Timetable";
+
+    trigger OnRun()
```
Comment (SAllhusen) [inline uniapp/src/page/UniTimetableMgt.Codeunit.al]: Please remove

File: uniapp/src/page/UniTimetableMgt.Codeunit.al [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        FillTimetableForTimeRange(UniTimetable, UniCourseHeader, UniCourseLineCount, 140000T, 150000T);
+        FillTimetableForTimeRange(UniTimetable, UniCourseHeader, UniCourseLineCount, 160000T, 170000T);
+    end;
+
+    local procedure FillTimetableForTimeRange(var UniTimetable: Record "Uni Timetable"; UniCourseHeader: Record "Uni Course Header"; UniCourseLineCount: Integer; StartTime: Time; EndTime: Time)
+    var
+        UniCourseLine: Record "Uni Course Line";
+        LineNo: Integer;
+    begin
+
```
Comment (SAllhusen) [inline uniapp/src/page/UniTimetableMgt.Codeunit.al]: Please remove empty line

Comment (ctm-fl) [reply uniapp/src/page/UniTimetableMgt.Codeunit.al]: Done.

File: uniapp/src/page/UniTimetableMgt.Codeunit.al (hunk 1 above)
Comment (ctm-fl) [reply uniapp/src/page/UniTimetableMgt.Codeunit.al]: Done.

**Review by SAllhusen — APPROVED** [review]

**Pull request merged**